system-test/spanner.ts

```diff
@@ -3606,22 +3565,10 @@ describe('Spanner', () => {
   });
 
   describe('SessionPool', () => {
-    const database = instance.database(generateName('database'));
-    const table = database.table('Singers');
-
-    before(async () => {
-      const [, operation] = await database.create({
-        schema: `
-            CREATE TABLE Singers (
-              SingerId STRING(1024) NOT NULL,
-              Name STRING(1024),
-            ) PRIMARY KEY(SingerId)`,
-      });
-      await operation.promise();
-    });
+    const table = DATABASE.table('Singers');
 
     it('should insert and query a row', done => {
-      const id = generateName('id');
+      const id = generateName('id0');
```

Comment:
Why id0?

Reply:
I think the query just grabs the lot of them, and then we pop off the most recently inserted, or at least that's what I was thinking when I wrote the other test that uses `id1` and `id2`. Was it returning the row from the Backups test when it was still just "id"? We should find a way to be certain not to depend on the other test in any way.

Reply:
Switched to check if the `row` on question is present in the `rows` array.